src/css/main.css

```diff
@@ -0,0 +1,41 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-image: url("../images/bg-image.jpg");
+  backdrop-filter: blur(5px);
+  background-size: cover;
+  padding: 0;
+  margin: 0;
+}
+
+#root {
+  width: 40%;
+  margin: 0 auto;
+  min-height: 94vh;
+  padding-top: 25px;
+}
+
+.table {
+  width: 100%;
+  min-height: 50vh;
+  padding-left: 10px;
+  background-color: #e7e4e4;
+  border-collapse: collapse;
+}
+
+td {
+  text-align: center;
+  border: #d1c4c4 solid 1px;
+  font-size: 16px;
+  font-weight: 600;
+  letter-spacing: 0.03em;
+  cursor: pointer;
+}
+
+td:last-child {
+  background-color: #ff6b00;
+  color: #fff;
+}
```

Comment:
- [ ] I think it would be better if you update some colors you used in your design by using the same ones as on the given **[wireframe](https://gitlab.com/microverse/guides/tse/code_review/code_review_guidelines_2_0/-/raw/master/react_redux/math_magicians/images/calculator.png)**.
```
  - Orange color: #f5913e;
  - Result background color(dar gray): #858693;
  - Numbers background color(light gray): #e0e0e0;
```

| Template Design | Your Design |
| --------------------- | ------------------- |
| ![Screenshot from 2022-09-07 16-52-11](https://user-images.githubusercontent.com/83042742/188909630-da97df89-80f0-48c3-a571-d2f6a06e0ef7.png) | ![Screenshot from 2022-09-07 16-52-27](https://user-images.githubusercontent.com/83042742/188909703-f07b7c84-22e5-42eb-a332-8e9b3adda759.png) |

Reply:
Hi @tresorsawasawa , 

Thank you so much for your constructive feedback. 
I tried f5913e for the orange background but it is lighter orange for that I used  ff6400.